webextension/css/popup.css

```diff
@@ -128,50 +318,45 @@ table.unstriped tbody tr {
   background: #dcdbdc;
 }
 
+/* Panel Footer */
 .panel-footer {
+  display: flex:
```

Comment:
Well, looks like that one colon/semicolon switchup was masking about 107 stylelint errors (mostly related to A-z violations and some spacing issues). Those should be easy for me (or anybody) to fix, but there are also about 48 `property-blacklist` errors where it doesn't like properties such as `width`, `margin`, `height`, `padding`, etc which may be tricker to solve. 

Alternatively, since we're so close to the deadline, we could disable the [`property-blacklist` rule](https://github.com/mozilla/testpilot-containers/blob/e1bb634f10eaf57d7febd7f7ceb90926dbe14e96/.stylelintrc#L9-L16) and file a new bug for the **Next** milestone to reassess that.